src/SplitPackages/CsprojFileBuilder.cs

```diff
@@ -114,39 +113,87 @@ public void AddImports(string fx, string import)
 
         public void Execute()
         {
-            var document = new JObject();
-            document["dependencies"] = JObject.FromObject(CreateDependenciesDictionary(_dependencies));
-            document["frameworks"] = JObject.FromObject(CreateFrameworksDictionary());
-            document["runtimes"] = JObject.FromObject(CreateRuntimesDictionary());
+            var document = new XDocument();
+            var root = new XElement("Project", new XAttribute("Sdk", "Microsoft.NET.Sdk"), new XAttribute("ToolsVersion", "15.0"));
```

Comment:
Nit: we don't need to add the ToolsVersion attribute anymore (was a recent change.)